src/metabase/integrations/ldap.clj

```diff
@@ -183,10 +183,10 @@
    (with-connection find-user username))
 
   ([conn username]
-   (when-let [{:keys [dn], :as result} (search conn username)]
-     (let [{fname (keyword (ldap-attribute-firstname))
-            lname (keyword (ldap-attribute-lastname))
-            email (keyword (ldap-attribute-email))}    result]
+   (when-let [{:keys [dn], :as result} (into (empty {}) (for [[key value] (search conn username)] [(keyword (.toLowerCase (name key))) value]))]
```

Comment:
I think we can do without the call to `empty` in this case, since you're using an empty map literal `{}` already. Or am I missing something?

Reply:
Makes sense to me. Will change that.